### ISSUE TITLE
Default Learning Rates

### ISSUE THREAD

**Issue by tylergiallanza** [PrincetonUniversity/PsyNeuLink#2015]
Currently, the default learning rate for the `add_backpropagation_learning_pathway` function of `Composition` (learning_rate=0.05) is different than the default learning rate for `AutodiffComposition` (learning_rate=0.001). 

There may be good reasons for this discrepancy, but it's a bit unnatural when trying to compare the results of learning in a Composition with and without autodiff. I think it may be beneficial to make these the same value to avoid any confusion.

**Comment by jdcpni** [comment]
Thanks.  Agreed.


jdc



On Apr 21, 2021, at 4:42 PM, tylergiallanza ***@***.******@***.***>> wrote:



Currently, the default learning rate for the add_backpropagation_learning_pathway function of Composition (learning_rate=0.05) is different than the default learning rate for AutodiffComposition (learning_rate=0.001).

There may be good reasons for this discrepancy, but it's a bit unnatural when trying to compare the results of learning in a Composition with and without autodiff. I think it may be beneficial to make these the same value to avoid any confusion.

**Comment by kmantel** [comment]
Which should be the default (or should both be changed to something else)?
